src/librustc_resolve/lib.rs

```diff
@@ -753,6 +753,9 @@ pub struct ModuleS<'a> {
     parent_link: ParentLink<'a>,
     def: Option<Def>,
 
+    // The node id of the closest normal module (`mod`) ancestor (including this module).
```

Comment:
So, for example for an item defined inside a function inside a module `foo`, this would be the id for `foo`? What is it used for? Evaluating `super` in paths, meaning of pub(restricted), is that all?